sdk/Trace/SimpleSpanProcessor.php

```diff
@@ -23,24 +28,38 @@ public function __construct(Exporter $exporter)
      */
     public function onStart(API\Span $span): void
     {
-        // nothing to do here
     }
 
     /**
      * @inheritDoc
      */
     public function onEnd(API\Span $span): void
     {
-        // @todo only spans with SampleFlag === true should be exported according to
-        // https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sdk-tracing.md#sampling
-        $this->exporter->export([$span]);
+        if ($this->running) {
```

Comment:
Should we address this todo in this PR or do you think it should be in a separate PR?